docs/fsharp/language-reference/casting-and-conversions.md

```diff
@@ -94,19 +94,13 @@ The following code illustrates the use of the `:>` and `:?>` operators. The code
 
 [!code-fsharp[Main](~/samples/snippets/fsharp/lang-ref-2/snippet4403.fs)]
 
-Because generic operators `downcast` and `upcast` rely on type inference to determine the argument and return type, in the above code, you can replace
-
-```fsharp
-let base1 = d1 :> Base1
-```
-
-with
+Because the generic operators `downcast` and `upcast` rely on type inference to determine the argument and return type, you can replace `let base1 = d1 :> Base1` in the previous code with:
```

Comment:
@gewarren Thank you for your help to fix this, but it would be better if the last code block is also added into this sentence as inline code. Otherwise, the translator will not know what follows after "with:" and it may cause difficulty/questions when translating this.

Reply:
@JasonCard Thanks, this is fixed now.